### PR TITLE
PLT-3642 Add PDF previewer

### PR DESCRIPTION
**Pull request by jwilander**
#### Summary
Add v1 of the ability to preview PDFs.

#### Ticket Link
https://mattermost.atlassian.net/browse/PLT-3642

#### Checklist
- [x] Has UI changes
- [x] Includes text changes and localization files updated

### CODE REVIEW

File: webapp/components/pdf_preview.jsx [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+// See License.txt for license information.
+
+import FileInfoPreview from './file_info_preview.jsx';
+
+import * as Utils from 'utils/utils.jsx';
+
+import loadingGif from 'images/load.gif';
+
+import React from 'react';
+import PDFJS from 'pdfjs-dist';
```
Comment (enahum) [inline webapp/components/pdf_preview.jsx]: I thought we were using all caps for constants only

Comment (jwilander) [inline webapp/components/pdf_preview.jsx]: I think imports would be the only exception, they were using `PDFJS` in all their documentation. 0/5

**Comment by enahum** [comment]
+1 other than my note on constants

**Comment by lfbrock** [comment]
+1

Tested posting multiple pdfs and single pdfs, tested with pdfs less than 5 pages and more than 5 pages. Didn't find any issue.

**Comment by coreyhulen** [comment]
+1

**Pull request merged**